client/my-sites/stats/stats-notices/all-notice-definitions.ts

```diff
@@ -64,6 +65,42 @@ const ALL_STATS_NOTICES: StatsNoticeType[] = [
 		},
 		disabled: false,
 	},
+	{
+		component: CommercialSiteUpgradeNotice,
+		noticeId: 'commercial_site_upgrade',
+		isVisibleFunc: ( {
```

Comment:
OK sure, makes sense. I have a few things in place to prevent accidental display but I think this would be a really good extra step to prevent any edge cases of it displaying.